aiida/backends/tests/cmdline/commands/test_group.py

```diff
@@ -210,3 +210,31 @@ def test_add_remove_nodes(self):
         result = self.cli_runner.invoke(group_remove_nodes, ['--force', '--clear', '--group=dummygroup1'])
         self.assertClickResultNoException(result)
         self.assertEqual(group.count(), 0)
+
+    def test_copy_existing_group(self):
+        """Test user is prompted to continue if destination group exists and is not empty"""
+        source_label = "source_copy_existing_group"
+        dest_label = "dest_copy_existing_group"
+
+        # Create source group with nodes
+        calc_s1 = orm.CalculationNode().store()
+        calc_s2 = orm.CalculationNode().store()
+        source_group = orm.Group(label=source_label).store()
+        source_group.add_nodes([calc_s1, calc_s2])
+
+        # Copy using `verdi group copy` - making sure all is successful
+        options = [source_label, dest_label]
+        result = self.cli_runner.invoke(group_copy, options)
+        self.assertClickResultNoException(result)
+        self.assertIn(
```

Comment:
It would be good to check that the destination group was actually created and it contains the nodes you expect by loading it through normal ORM API. It is true that we are not testing the actually ORM functionality, but it could happen that due to a developer's mistake, the proper logic to copy over nodes to the new group is never called.

Reply:
Done